nex/secure.go

```diff
@@ -15,16 +15,22 @@ import (
 	_ "github.com/PretendoNetwork/nex-protocols-go"
 )
 
-func StartNEXServer() {
-	globals.NEXServer = nex.NewServer()
-	globals.NEXServer.SetFragmentSize(900)
-	globals.NEXServer.SetPRUDPVersion(0)
-	globals.NEXServer.SetKerberosKeySize(16)
-	globals.NEXServer.SetKerberosPassword(os.Getenv("PN_FRIENDS_CONFIG_KERBEROS_PASSWORD"))
-	globals.NEXServer.SetPingTimeout(20) // Maybe too long?
-	globals.NEXServer.SetAccessKey("ridfebb9")
+func StartSecureServer() {
+	globals.SecureServer = nex.NewServer()
+	globals.SecureServer.SetFragmentSize(900)
+	globals.SecureServer.SetPRUDPVersion(0)
+	globals.SecureServer.SetKerberosKeySize(16)
+	globals.SecureServer.SetKerberosPassword(globals.KerberosPassword)
+	globals.SecureServer.SetPingTimeout(20) // Maybe too long?
+	globals.SecureServer.SetAccessKey("ridfebb9")
+	globals.AuthenticationServer.SetPRUDPProtocolMinorVersion(0) // TODO: Figure out what to put here
```

Comment:
We don't have to apply this here, the NEX server will replicate the PRUDP minor version of the client.

This is applied on some games because they use PRUDP minor version >=4, which isn't compatible with our servers (and we don't know the difference with the previous versions).

```suggestion
```

Reply:
Removed function call